app/assets/javascripts/discourse/app/components/d-editor.hbs

```diff
@@ -40,15 +48,18 @@
 
       <ConditionalLoadingSpinner @condition={{this.loading}} />
```

Comment:
Should we display a loading spinner when `editorComponent` is null? (i.e. rich editor is loading)

Reply:
I did have an 'or' here at some point, but I felt it was so fast it kind of worsened the UX (even with network throttling).

Reply:
Fair enough. In some places we have logic like "if loading for more than 500ms, show the loading spinner". But I'm fine for us to leave as-is unless it becomes a real problem :+1: